sdks/python/apache_beam/examples/inference/pytorch_image_classification.py

```diff
@@ -72,6 +73,7 @@ def parse_known_args(argv):
   """Parses args for the workflow."""
   parser = argparse.ArgumentParser()
   parser.add_argument(
+      '--input_file',
```

Comment:
does this mean it accepts either input_file or input as an argument?

Reply:
Removed the --input_file